food-truck-frontend/src/components/login/LoginComponent.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import Form from "../Form";
+import {AxiosResponse} from 'axios';
+import {Grid, TextField, Typography} from '@material-ui/core'
+import {UserData} from "../../redux/user/UserReducer";
+import NextLink from 'next/link'
+import LinkButton from "../layout/LinkButton";
+import Router from "next/router";
+
+type State = {
+  resultText: string;
+  data: UserData;
+  loadUserFromBackend: () => Promise<void>;
+}
+
+interface LoginProps {
+  data: UserData;
```

Comment:
I'm not seeing this prop being used anywhere, or its reflection in the state.

Also worth noting, I'm fairly sure I read once that having a state variable that's supposed to reflect a props variables is an antipattern (can't find the article though, sorry). Thus, if this is needed, it should probably be kept in one or the other, not both.

Reply:
I think some of this code was already there when I touched the file. Either way, I need `loadUserFromBackend` so I think the `data` field will still get passed to the component by redux